test-java-labs.iml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
+  <component name="ExternalSystem" externalSystem="Maven" />
```

Comment:
what is this?

Reply:
I do not remember doing this, if did it is by mistake. Have no idea.

Reply:
please draw a UML diagram and share